service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1907,13 +1907,15 @@ public TariffsForLocationDto getTariffForOrder(Long id) {
     public Set<String> getAllAuthorities(String email) {
         Employee employee = employeeRepository.findByEmail(email)
             .orElseThrow(() -> new NotFoundException(EMPLOYEE_DOESNT_EXIST));
-        return userRemoteClient.getAllAuthorities(employee.getEmail());
+        Set<String> authorities = userRemoteClient.getAllAuthorities(employee.getEmail());
+        if (authorities.isEmpty()) {
```

Comment:
please add test for this case

Reply:
Is it not a valid case when an employee does not have any authorities?
Do we set some authorities when we create employees? 

And according to your [changes](https://github.com/ita-social-projects/GreenCityUBS/pull/947/files#diff-7177ebba715f0c0fa9b920a0ac8f2501274d86382af783b2080cd07d7be2c66fR62), we will have empty authorities list if some error happens when we call `/user/get-all-authorities` endpoint
In this case, the employee could have some authorities but we could not get them and your message `COULD_NOT_RETRIEVE_EMPLOYEE_AUTHORITY` will not be valid